### ISSUE TITLE
Classify EM parameters depending on its class

### ISSUE THREAD

**Issue by antolinos** [antolinos/em-model#1]
There are some parameters that seem to belong to either movie or acquisition however today most of them are in the same table.
List of parameters that need classification:
```
magnification (unit: X)
total_absorbed_dose (Unit: e-/A^2 for EM)
binning (1 or 2. Number of pixels to process as 1. (Use mean value.)
particle_diameter (Unit: nm)
box_size_ctf (Unit: pixels)
min_resolution (Unit: A)
min_defocus (Unit: A)
max_defocus (Unit: A)
defocus_step_size (Unit: A)
amount_astigmatism (Unit: A)
extract_size (Unit: pixels)
bg_radius (Unit: nm)
voltage (Unit: kV)
obj_aperture (Unit: um)
c1aperture (Unit: um)
c2aperture (Unit: um)
c3aperture (Unit: um)
c1lens (Unit: %)
c2lens (Unit: %)
c3lens (Unit: %)
FFT of the raw micrograph
FFT of the (motion?) corrected micrograph
full path to a .png of the summed (motion?) corrected micrograph


```

Proposed hierarchy:
![image](https://user-images.githubusercontent.com/2901169/28267238-6e0af3fa-6af9-11e7-8368-c4f60d116b37.png)

**Comment by antolinos** [comment]
First iteration, information about particles are omitted. 

**Two more tables have been created:**
```
Micrograph 1 -----  N Movie
CTF 1 ---- micrograph
```

Comments about some params:
```
box_size_ctf to be saved within the output files
min_resolution needs a definition (It is probably coming from MX)
```

Data model is now like:
![image](https://user-images.githubusercontent.com/2901169/28278977-46d1f434-6b1f-11e7-91cf-8d15770e73a1.png)

**Comment by antolinos** [comment]
Second iteration:
![image](https://user-images.githubusercontent.com/2901169/28317222-501c32f0-6bc6-11e7-96f7-65227c915456.png)

**Comment by antolinos** [comment]
3rd Iteration:
![image](https://user-images.githubusercontent.com/2901169/28578762-f2bc0320-715a-11e7-8801-2a90d2c6e782.png)

**Comment by stufisher** [comment]
Hello all,

I have taken Alex's model above, input from our EM and SciSoft staff here at DLS, and input from EM people on the EPN campus and mapped this against the existing ISPyB database structure. I think what we have is pretty complete. I have tried to annotate what i can, if anything doesnt make sense please shout. There will probably be some further minor changes to this. Tables in yellow will not be implemented at this time as it seems the software architecture is moving very fast.

Ta
Stu

![em_ispyb_model](https://user-images.githubusercontent.com/4463399/29261785-a6daf4f8-80d1-11e7-9671-9e15fa070d54.png)

**Comment by antolinos** [comment]
Thanks for this!
Should it be moved to https://github.com/ispyb/ispyb-database-modeling?

**Comment by stufisher** [comment]
Yep sounds good. I am on site if you want to have a chat about this too
